src/Yargs.fs

```diff
@@ -50,17 +50,17 @@ module yargs =
         abstract requiresArg: key: string -> Argv
         abstract requiresArg: keys: ResizeArray<string> -> Argv
         abstract describe: key: U2<string, ResizeArray<string>> * description: string -> Argv
-        abstract describe: descriptions: obj -> Argv
+        abstract describe: descriptions: ArgvDescribeDescriptions -> Argv
         abstract option: key: string * options: Options -> Argv
-        abstract option: options: obj -> Argv
+        abstract option: options: ArgvOptionOptions -> Argv
         abstract options: key: string * options: Options -> Argv
-        abstract options: options: obj -> Argv
-        abstract usage: message: string * ?options: obj -> Argv
-        abstract usage: ?options: obj -> Argv
+        abstract options: options: ArgvOptions -> Argv
```

Comment:
I the method named was the same as the param name, I kept it `ArgvOptions` instead of `ArgvOptionsOptions`.